File: vibe-lang/benchmark/output/lua-love/openai/color_pulse.lua
local timer = 0.0

function love.load()
    love.window.setMode(800, 600)
    love.window.setTitle("Pulsing Circles Grid")
end

function love.update(dt)
    timer = timer + dt
end

function love.draw()
    for row = 0, 3 do
        for col = 0, 4 do
            local cx = 100.0 + col * 160.0
            local cy = 100.0 + row * 150.0
            local phase = (row * 5 + col) * 0.5
            local radius = 20.0 + 15.0 * math.sin(timer * 3.0 + phase)
            love.graphics.circle("fill", cx, cy, radius)
        end
    end
    love.graphics.print("Time: " .. tostring(math.floor(timer)), 10.0, 10.0)
end

File: benchmark/output/python-pygame/openai/color_pulse.py
import pygame
import sys
import math

pygame.init()
screen = pygame.display.set_mode((800, 600))
pygame.display.set_caption("Pulsing Circles Grid")
clock = pygame.time.Clock()

timer = 0.0

running = True
while running:
    dt = clock.tick(60) / 1000.0
    timer += dt

    for event in pygame.event.get():
        if event.type == pygame.QUIT:
            running = False

    screen.fill((0, 0, 0))

    for row in range(0, 4):
        for col in range(0, 5):
            cx = 100.0 + float(col) * 160.0
            cy = 100.0 + float(row) * 150.0
            phase = float(row * 5 + col) * 0.5
            radius = 20.0 + 15.0 * math.sin(timer * 3.0 + phase)
            pygame.draw.circle(screen, (255, 255, 255), (int(cx), int(cy)), int(radius))

    font = pygame.font.Font(None, 36)
    text = font.render("Time: " + str(int(timer)), True, (255, 255, 255))
    screen.blit(text, (10.0, 10.0))

    pygame.display.flip()

pygame.quit()
sys.exit()